docs/source/notebooks/clv/clv-quickstart.ipynb

```diff
@@ -0,0 +1,2270 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import arviz as az
</pre><p>Shall we use an uniffyed style as </p><p> </p><pre class="ql-syntax" spellcheck="false">az.style.use("arviz-darkgrid")
%config InlineBackend.figure_format = "retina" # nice looking plots
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>We should. I think I removed it accidentally</p>